src/lib/marketplaces/helper.ts

```diff
@@ -39,7 +39,11 @@ export function parseNFTSaleOnTx(
     return null;
   }
 
-  if (!txResp.meta.logMessages[0].includes(marketplace.programId)) {
+  if (
+    !txResp.meta.logMessages.filter((msg) =>
+      msg.includes(marketplace.programId)
+    ).length
```

Comment:
Setting this to a `const` might make it a bit easier to parse. Takes me a minute to read it